vivisect/tests/testpe.py

```diff
@@ -16,6 +16,23 @@ def setUpClass(cls):
         cls.vw_psexec.loadFromFile(cls.psexec_fn)
         cls.vw_psexec.analyze()
 
+    def test_xrefs_types(self):
+        vw = self.vw_psexec
+        for xfrom, xto, xtype, xflags in vw.getXrefs():
+            self.assertTrue(isinstance(xfrom, int))
+            self.assertTrue(isinstance(xto, int))
+            self.assertTrue(isinstance(xtype, int))
+            self.assertTrue(isinstance(xflags, int))
+
+    def test_loc_types(self):
+        vw = self.vw_psexec
+        for lva, lsize, ltype, linfo in vw.getLocations():
+            self.assertTrue(isinstance(lva, int))
+            self.assertTrue(isinstance(lsize, int))
```

Comment:
same here.  what's the value?

Reply:
It's a data conformance thing. All location tuples and xrefs have a general structure that users expect, and this is codifying that into a proper test (I actually made this test to mirror the xref one).